packages/hotpotqa/src/aviary/envs/hotpotqa/env.py

```diff
@@ -340,6 +341,7 @@ def finish(self, answer: str) -> str:
 
         self.state.answer = answer
         self.state.reward += self.calculate_reward(answer)
+        self.state.last_action = "Finish"
```

Comment:
Does this relate to the tool name? We should maybe encapsulate this into a method of state

At least we should use `tool.info.name` over redefining the string